Makefile

```diff
@@ -3,3 +3,7 @@ default:
 
 test:
 	GO111MODULE=on go test ./...
+
+check:
+	GO111MODULE=off go get golang.org/x/lint/golint
```

Comment:
How about using `golangci-lint`?

Reply:
Maybe later? I want it to work first and then improve.